### ISSUE TITLE
graphite-web might be incompatible with whitenoise 4

### ISSUE THREAD

**Issue by ybizeul** [graphite-project/graphite-web#2332]
```
[Fri Aug 17 15:44:30.702758 2018] [wsgi:error] [pid 20006] [remote 127.0.0.1:21300] mod_wsgi (pid=20006): Target WSGI script '/opt/graphite/conf/graphite.wsgi' cannot be loaded as Python module.
[Fri Aug 17 15:44:30.702793 2018] [wsgi:error] [pid 20006] [remote 127.0.0.1:21300] mod_wsgi (pid=20006): Exception occurred processing WSGI script '/opt/graphite/conf/graphite.wsgi'.
[Fri Aug 17 15:44:30.702813 2018] [wsgi:error] [pid 20006] [remote 127.0.0.1:21300] Traceback (most recent call last):
[Fri Aug 17 15:44:30.702840 2018] [wsgi:error] [pid 20006] [remote 127.0.0.1:21300]   File "/opt/graphite/conf/graphite.wsgi", line 4, in <module>
[Fri Aug 17 15:44:30.702886 2018] [wsgi:error] [pid 20006] [remote 127.0.0.1:21300]     from graphite.wsgi import application
[Fri Aug 17 15:44:30.702894 2018] [wsgi:error] [pid 20006] [remote 127.0.0.1:21300]   File "/opt/graphite/webapp/graphite/wsgi.py", line 29, in <module>
[Fri Aug 17 15:44:30.702907 2018] [wsgi:error] [pid 20006] [remote 127.0.0.1:21300]     from whitenoise.django import DjangoWhiteNoise
[Fri Aug 17 15:44:30.702914 2018] [wsgi:error] [pid 20006] [remote 127.0.0.1:21300]   File "/usr/local/lib/python2.7/dist-packages/whitenoise/django.py", line 2, in <module>
[Fri Aug 17 15:44:30.702925 2018] [wsgi:error] [pid 20006] [remote 127.0.0.1:21300]     '\\n\\n'
[Fri Aug 17 15:44:30.702942 2018] [wsgi:error] [pid 20006] [remote 127.0.0.1:21300] ImportError: 
[Fri Aug 17 15:44:30.702945 2018] [wsgi:error] [pid 20006] [remote 127.0.0.1:21300] 
[Fri Aug 17 15:44:30.702948 2018] [wsgi:error] [pid 20006] [remote 127.0.0.1:21300] Your WhiteNoise configuration is incompatible with WhiteNoise v4.0
[Fri Aug 17 15:44:30.702950 2018] [wsgi:error] [pid 20006] [remote 127.0.0.1:21300] This can be fixed by following the upgrade instructions at:
[Fri Aug 17 15:44:30.702953 2018] [wsgi:error] [pid 20006] [remote 127.0.0.1:21300] http://whitenoise.evans.io/en/stable/changelog.html#v4-0
```

This fixed it :

```
root@nabox:/opt/graphite# pip uninstall whitenoise
/usr/local/lib/python2.7/dist-packages/cffi/model.py:531: UserWarning: 'point_conversion_form_t' has no values explicitly defined; guessing that it is equivalent to 'unsigned int'
  % self._get_c_name())
Uninstalling whitenoise-4.0:
  Would remove:
    /usr/local/lib/python2.7/dist-packages/whitenoise-4.0.dist-info/*
    /usr/local/lib/python2.7/dist-packages/whitenoise/*
Proceed (y/n)? y
  Successfully uninstalled whitenoise-4.0
root@nabox:/opt/graphite# pip install "whitenoise<4"
/usr/local/lib/python2.7/dist-packages/cffi/model.py:531: UserWarning: 'point_conversion_form_t' has no values explicitly defined; guessing that it is equivalent to 'unsigned int'
  % self._get_c_name())
Collecting whitenoise<4
  Downloading https://files.pythonhosted.org/packages/0c/58/0f309a821b9161d0e3a73336a187d1541c2127aff7fdf3bf7293f9979d1d/whitenoise-3.3.1-py2.py3-none-any.whl
Installing collected packages: whitenoise
Successfully installed whitenoise-3.3.1
```

**Comment by deniszh** [comment]
Yes, fixed in https://github.com/graphite-project/graphite-web/pull/2331 - just pinning to latest 3.x for now

**Comment by deniszh** [comment]
Fixed in 1.1.4
